test/jdk/javax/swing/JMenuItem/TestImageIconWithJRadioButtonMenuItemAndJCheckBoxMenuItem.java

```diff
@@ -0,0 +1,141 @@
+        Second radiobutton menuitem is unselected with imageicon.
+        Third radiobutton menuItem is unselected without imageicon.
+
+        First checkbox menuitem is selected with imageicon.
+        Second checkbox menuitem is unselected with imageicon.
+        Third checkbox menuItem is unselected without imageicon.
+
+        Verify that for first JRadioButtonMenuItem with image icon,
+        a bullet is shown alongside the imageicon and
+        for first JCheckBoxMenuItem with imageicon
+        a checkmark is shown alongside the image icon.
+
+        If bullet and checkmark is shown, test passes else fails.""";
+
+    public static void main(String[] args) throws Exception {
+        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+        PassFailJFrame.builder()
+                .title("JRadioButtonMenuItem Instructions")
+                .instructions(INSTRUCTIONS)
+                .columns(60)
+                .testUI(
+                 TestImageIconWithJRadioButtonMenuItemAndJCheckBoxMenuItem::doTest)
+                .build()
+                .awaitAndCheck();
+    }
+
+    public static JFrame doTest() {
+        BufferedImage img = new BufferedImage(16, 16, BufferedImage.TYPE_INT_ARGB);
+        Graphics g = img.getGraphics();
+        g.setColor(Color.red);
+        g.fillRect(0, 0, img.getWidth(), img.getHeight());
+        g.dispose();
+
+        JFrame frame = new JFrame("RadioButtonWithImageIcon");
+        ImageIcon imageIcon1 = new ImageIcon(img);
+        AbstractButton button1 = new JRadioButtonMenuItem("JRadioButtonMenuItem 1",
+                imageIcon1);
+        button1.setSelected(true);
+        AbstractButton button2 = new JRadioButtonMenuItem("JRadioButtonMenuItem 2", imageIcon1);
+        AbstractButton button3 = new JRadioButtonMenuItem("JRadioButtonMenuItem 3");
+
+        ButtonGroup buttonGroup = new ButtonGroup();
+        buttonGroup.add(button1);
+        buttonGroup.add(button2);
+        buttonGroup.add(button3);
+
+        AbstractButton check1 = new JCheckBoxMenuItem("JCheckBoxMenuItem 1",
+                imageIcon1);
+        check1.setSelected(true);
+        AbstractButton check2 = new JCheckBoxMenuItem("JCheckBoxMenuItem 2", imageIcon1);
+        AbstractButton check3 = new JCheckBoxMenuItem("JCheckBoxMenuItem 3");
+        ButtonGroup buttonGroup1 = new ButtonGroup();
+        buttonGroup1.add(check1);
+        buttonGroup1.add(check2);
+        buttonGroup1.add(check3);
+
+        JMenu topLevel = new JMenu("Menus");
+
+        topLevel.add(button1);
+        topLevel.add(button2);
+        topLevel.add(button3);
+
+        topLevel.addSeparator();
+
+        topLevel.add(check1);
+        topLevel.add(check2);
+        topLevel.add(check3);
+
+        JMenuBar menuBar = new JMenuBar();
+        menuBar.add(topLevel);
+
+        frame.setJMenuBar(menuBar);
+        frame.setSize(300, 300);
+        frame.setLocationRelativeTo(null);
```

Comment:
`setLocationRelativeTo` is redundant, the location of the frame is controlled by `PassFailJFrame`.